keyboards/dichotomy/dichotomy.h

```diff
@@ -1,15 +1,18 @@
-#ifndef DICHOTEMY_H
-#define DICHOTEMY_H
+#ifndef DICHOTOMY_H
+#define DICHOTOMY_H
 
+#include QMK_KEYBOARD_H
+#include "report.h"
+#include "pointing_device.h"
 #include "quantum.h"
 #include "matrix.h"
 #include "backlight.h"
 #include <stddef.h>
 
-#define red_led_off   PORTF |= (1<<5)
-#define red_led_on    PORTF &= ~(1<<5)
-#define blu_led_off   PORTF |= (1<<4)
-#define blu_led_on    PORTF &= ~(1<<4)
+#define red_led_off   PORTF |= (1<<6)
```

Comment:
Sorry for keep on finding things to change, but could you you include paranthesis so they can be called programatically? 

Eg: 
```suggestion
#define red_led_off()   PORTF |= (1<<6)
```

So that below, you'd use:
```c
#define set_led_cyan()    red_led_off(); grn_led_on();  blu_led_on()
```